service/health/searcher/response.go

```diff
@@ -1,5 +1,16 @@
 package searcher
 
+// GeneralStatus holds data about the overall health/status of a cluster.
+type GeneralStatus struct {
+	Health    string `json:"health"`
+	Creating  bool   `json:"creating"`
+	Upgrading bool   `json:"upgrading"`
+	Deleting  bool   `json:"deleting"`
```

Comment:
Isn't it better to replace `Creating` & `Upgrading` & `Deleting` & `Normal`, with `State` or `Status` and have some constants?

Also `Normal` looks weird. Do we have some design for that? In the CR we have "created", "updated` etc.

Reply:
I was trying to stick as a close to https://docs.google.com/spreadsheets/d/1nx2VdCwDEP2e5DlobjtFr7ANZTDso6_xDvzrs6sJu0A/edit#gid=0 as possible. I don't know what normal means exactly myself. My initial approach was to say a cluster is normal if it's not creating/upgrading/deleting.

Reply:
Since the states are mutually exclusive, I think this is a good place for an enum type. This change is committed now.

Reply:
So you have that documented cool. Yeah I figured but still normal seems weird to me. Maybe you can bring the "state" thing to your team. I think it's more sensible approach than ever growing list of mutual exclusive booleans.

Reply:
So we have agreement :D